Makefile

```diff
@@ -82,7 +82,7 @@ GINKGO_FOCUS ?=
 GINKGO_SKIP ?=
 GINKGO_NODES ?= 1
 GINKGO_TIMEOUT ?= 2h
-GINKGO_POLL_PROGRESS_AFTER ?= 10m
+GINKGO_POLL_PROGRESS_AFTER ?= 60m
```

Comment:
What about bumping also the progress interval to 5 minutes?
In the end, if I got this right, we are using this feature only to check that the jobs are not stuck, so we don't really need a stack trace every minute

Reply:
Sounds good

Reply:
Done